src/gui/properties/peerlistwidget.cpp

```diff
@@ -51,464 +51,456 @@
-        selectedPeers << "[" + ip + "]:" + myport;
-    else // IPv4
-        selectedPeers << ip + ":" + myport;
-  }
-  QApplication::clipboard()->setText(selectedPeers.join("\n"));
+    QModelIndexList selectedIndexes = selectionModel()->selectedRows();
+    QStringList selectedPeers;
+    foreach (const QModelIndex &index, selectedIndexes) {
+        int row = m_proxyModel->mapToSource(index).row();
+        QString ip = m_listModel->data(m_listModel->index(row, PeerListDelegate::IP_HIDDEN)).toString();
+        QString myport = m_listModel->data(m_listModel->index(row, PeerListDelegate::PORT)).toString();
+        if (ip.indexOf(".") == -1) // IPv6
+            selectedPeers << "[" + ip + "]:" + myport;
+        else // IPv4
+            selectedPeers << ip + ":" + myport;
+    }
+    QApplication::clipboard()->setText(selectedPeers.join("\n"));
 }
 
-void PeerListWidget::clear() {
-  qDebug("clearing peer list");
-  m_peerItems.clear();
-  m_peerAddresses.clear();
-  m_missingFlags.clear();
-  int nbrows = m_listModel->rowCount();
-  if (nbrows > 0) {
-    qDebug("Cleared %d peers", nbrows);
-    m_listModel->removeRows(0,  nbrows);
-  }
+void PeerListWidget::clear()
+{
+    qDebug("clearing peer list");
+    m_peerItems.clear();
+    m_peerAddresses.clear();
+    m_missingFlags.clear();
+    int nbrows = m_listModel->rowCount();
+    if (nbrows > 0) {
+        qDebug("Cleared %d peers", nbrows);
+        m_listModel->removeRows(0,  nbrows);
+    }
 }
 
-void PeerListWidget::loadSettings() {
-  header()->restoreState(Preferences::instance()->getPeerListState());
+void PeerListWidget::loadSettings()
+{
+    header()->restoreState(Preferences::instance()->getPeerListState());
 }
 
-void PeerListWidget::saveSettings() const {
-  Preferences::instance()->setPeerListState(header()->saveState());
+void PeerListWidget::saveSettings() const
+{
+    Preferences::instance()->setPeerListState(header()->saveState());
 }
 
-void PeerListWidget::loadPeers(BitTorrent::TorrentHandle *const torrent, bool force_hostname_resolution) {
-  if (!torrent) return;
-
-  QList<BitTorrent::PeerInfo> peers = torrent->peers();
-  QSet<QString> old_peers_set = m_peerItems.keys().toSet();
-
-  foreach (const BitTorrent::PeerInfo &peer, peers) {
-    BitTorrent::PeerAddress addr = peer.address();
-    if (addr.ip.isNull()) continue;
-
-    QString peer_ip = addr.ip.toString();
-    if (m_peerItems.contains(peer_ip)) {
-      // Update existing peer
-      updatePeer(peer_ip, torrent, peer);
-      old_peers_set.remove(peer_ip);
-      if (force_hostname_resolution && m_resolver) {
-        m_resolver->resolve(peer_ip);
-      }
-    } else {
-      // Add new peer
-      m_peerItems[peer_ip] = addPeer(peer_ip, torrent, peer);
-      m_peerAddresses[peer_ip] = addr;
-      // Resolve peer host name is asked
-      if (m_resolver)
-        m_resolver->resolve(peer_ip);
+void PeerListWidget::loadPeers(BitTorrent::TorrentHandle *const torrent, bool force_hostname_resolution)
```

Comment:
Please change:
    force_hostname_resolution => forceHostnameResolution
    old_peers_set => oldPeersSet
    peer_ip => peerIp
...and so on.
We prefer camelCased names in new code (and if you fixing coding style).